Winapp2.ini

```diff
@@ -8765,20 +8760,19 @@ FileKey1=%AppData%\IObit\Advanced SystemCare *|*.log
 FileKey2=%AppData%\IObit\Advanced SystemCare *\Boottime|*.log
 FileKey3=%AppData%\IObit\Advanced SystemCare *\EmptyFolder|*.*|RECURSE
 FileKey4=%AppData%\IObit\Advanced SystemCare *\Log|*.*|RECURSE
-FileKey5=%CommonAppData%\IObit\Advanced SystemCare*\Log|*.*|REMOVESELF
+FileKey5=%CommonAppData%\IObit\Advanced SystemCare *\Log|*.*|REMOVESELF
 FileKey6=%CommonAppData%\IObit\ASCDownloader|*.log
 FileKey7=%LocalAppData%\VirtualStore\Program Files*\IObit\Advanced SystemCare *|*.cab_Temp|REMOVESELF
 FileKey8=%LocalAppData%\VirtualStore\Program Files*\IObit\Advanced SystemCare *|*.log;*.txt|RECURSE
 FileKey9=%LocalAppData%\VirtualStore\Program Files*\IObit\Advanced SystemCare *\ASCServiceLog|*.*|RECURSE
-FileKey10=%LocalAppData%\VirtualStore\Program Files*\IObit\LiveUpdate|*.log
-FileKey11=%LocalAppData%\VirtualStore\ProgramData\IObit\Advanced SystemCare*\Log|*.*|RECURSE
-FileKey12=%LocalAppData%\VirtualStore\ProgramData\IObit\ASCDownloader|*.log
-FileKey13=%ProgramFiles%\IObit\Advanced SystemCare *|*.cab_Temp|REMOVESELF
-FileKey14=%ProgramFiles%\IObit\Advanced SystemCare *|*.log;*.txt|RECURSE
-FileKey15=%ProgramFiles%\IObit\Advanced SystemCare *\ASCServiceLog|*.*|RECURSE
-FileKey16=%SystemDrive%\Users\Default\AppData\Roaming\IObit\Advanced SystemCare *|*.log
-FileKey17=%WinDir%\System32\config\systemprofile\AppData\Roaming\IObit\Advanced SystemCare *|*.log
-FileKey18=%WinDir%\SysWOW64\config\systemprofile\AppData\Roaming\IObit\Advanced SystemCare *|*.log
+FileKey10=%LocalAppData%\VirtualStore\ProgramData\IObit\Advanced SystemCare *\Log|*.*|RECURSE
+FileKey11=%LocalAppData%\VirtualStore\ProgramData\IObit\ASCDownloader|*.log
+FileKey12=%ProgramFiles%\IObit\Advanced SystemCare *|*.cab_Temp|REMOVESELF
```

Comment:
This line looks as though it will delete the %ProgramFiles% directory for Advanced SystemCare. Am I reading this correctly?

Reply:
Theoretical, REMOVESELF means: if the directory is empty after deleting the \*.cab_Temp(s) then remove it. But the directory won't be empty.

I assume that the contributor of FileKey 12 wants to delete folders instead of files. The correct FileKey probably should be "%ProgramFiles%\IObit\Advanced SystemCare \*\\*.cab_Temp|\*.\*|REMOVESELF".

Reply:
I have tested and confirmed that REMOVESELF wont delete a non-empty folder, so this change shouldn't pose a threat to the program files directory.